### PR TITLE
docs: Edit the  v3 / admin / platforms / k3s  documentation

### PR DESCRIPTION
**Pull request by osamamagdy**
# Description
After trying to install Jenkins-x with the k3s approach I came across some problems that may be solved by changing commands in the documentation
# What did you change?
1) Explaining how to export the needed env variables permanently in the .bashrc file as you will be required to open multiple terminals
2) Copying the Configuration of the `/etc/rancher/k3s/k3s.yaml` to the `./kube/config` and changing the permissions if needed
3) Remainder to use v1.21 if installing k3s from their installation guide
4) Add docker installation and post-installation link from the official documentation
5) Add the step of installing the jx 3 binaries from the docs before running the jx admin operator command
6) If the jx secrets weren't generated, make a direct push through the GitHub repository itself, not the `git push` command (I don't actually know why this solved the problem with me but it may be an issue of git.

### CODE REVIEW

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 1]
```diff
@@ -34,26 +34,47 @@ If you dont have an existing k3s cluster, you can install one by running:
...
+```
+
+**Optional**
+
+- If the above method didn't work, copy the configurations to the ~/.kube/config instead (if you don't have any other clusters, that should be fine)
+```bash
+sudo rm ~/.kube/k3s-config  #to make k3s uses ~/.kube/config
+sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/config
+```
+- If still got permession denied
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:62]: typo: `permession`

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 2]
```diff
@@ -34,26 +34,47 @@ If you dont have an existing k3s cluster, you can install one by running:
...
+Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options but keep in mind we don't support Kubernetes 1.22+ yet
 
 #### Vault
 
 Install vault cli.
 Refer to the [vault docs](https://www.vaultproject.io/docs/install) on how to install vault for your platform.
 
 Make sure you have vault running in a docker container with kubernetes auth enabled.
-
+You need to install [docker](https://docs.docker.com/engine/install/) and [manage it as a non root user](https://docs.docker.com/engine/install/linux-postinstall/)
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:77]: Let's add this as a separate pre-req.
So now there are 3 pre-reqs:
- k3s
- docker
- vault

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 3]
```diff
@@ -74,7 +96,7 @@ Note: If you get the error `Error enabling kubernetes auth: Post "https://127.0.
 - This bot user needs to have write permission to write to any git repository used by Jenkins X. This can be done by adding the bot user to the git organisation level or individual repositories as a collaborator Add the new bot user to your Git Organisation, for now give it Owner permissions, we will reduce this to member permissions soon.
 
 ### Jenkins X v3 installation
-
+- Make sure you have installed [jx 3.x binary](/v3/guides/jx3/) and put it on your `$PATH` as the `jx admin operator` will be used
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:99]: Also add as pre-req, and link to the cli installation page: https://jenkins-x.io/v3/admin/setup/jx3/

File: content/en/v3/admin/platforms/k3s/_index.md [hunk 4]
```diff
@@ -160,6 +185,7 @@ This job will create the secrets in vault which will be used by external secrets
 
 - To verify the job succeeded, run `jx admin log`
 - To verfiy the secrets were created, run `kubectl get es -A` and `jx secret verify`
+- If this didn't work try and repeat the steps but commit your dummy changes through github repository directly without
```
Comment (ankitm123) [inline content/en/v3/admin/platforms/k3s/_index.md:188]: Seems like this statement is incomplete, also the boot job runs 5 times on failure, so my guess is that in ur case, those were still old jobs, but nevertheless I think it's ok to add this.

**Review by ankitm123 — COMMENTED** [review]
minor comments, thanks for doing this :+1:

**Comment by osamamagdy** [comment]
@ankitm123 All the requested changes are done now

**Pull request merged**